core/templates/domain/topic/subtopic.model.ts

```diff
@@ -92,20 +92,22 @@ export class Subtopic {
   }
 
   validate(): string[] {
-    var issues = [];
+    let issues = [];
```

Comment:
```suggestion
    const issues: string[] = [];
```
Lets be a little more explicit. issues here is assumed to be of type any[]. (Are we covering this file?) If not then I am fine.

Reply:
Yes,covered now but wouldn't making it constant a wrong thing since it's value is changing

Reply:
The contants you are talking about would be like `const ISSUES` and not `const issues`. But you've brought up a valid point. I don't know if everyone follows this. @vojtechjelinek and @seanlip Do you mind sharing your opinion on this. So basically, issues here is not reassigned. But new values are pushed into the array. The cases where the values don't change at all, that variable name is in all caps.

Reply:
It's a good question -- I'm inclined for "issues" here not to be in all caps.

The thing I'm wondering is whether it should be a constant. I feel like it shouldn't be (conceptually) and that we should reserve "constant" for things that don't change. I wonder what @vojtechjelinek thinks.

Reply:
I agree with Sean.